src/components/select-button/index.tsx

```diff
@@ -0,0 +1,40 @@
+import React from 'react';
+import './index.css';
+
+interface SelectButtonProps {
+  label?: string;
+  data:string;
+  color?: 'black' | 'white';
+  backgroundColor?: 'blue' | 'green'; 
+  SetActive: () => void;
+  isActive?: boolean;
+}
+
+const SelectButton: React.FC<SelectButtonProps> = ({
+  label,
+  data,
+  color,
+  backgroundColor,
+  SetActive,
+  isActive = false,
+}: SelectButtonProps) => {
+  const handleButtonClick = () => {
+    SetActive();
+  };
+
+  const buttonStyle: React.CSSProperties = {
+    color: isActive ? 'white' : color || 'black',
+    backgroundColor: isActive ? '#527cf8' : backgroundColor || '#dcdcdc',
+  };
+
+  return (
+    <button className='styled-select-button'
+      onClick={handleButtonClick}
+      style={buttonStyle}
+    >
+      {data}
+    </button>
+  );
```

Comment:
ilk button ve click değil aslında ilk div olmalı çünkü list gibi düşünebilirsin

{data}? data mapping olmalı

["Todo", "In progress", "Muhammed"] bu gibi olmalı